server/src/main/kotlin/net/horizonsend/ion/server/features/client/VoidNetwork.kt

```diff
@@ -0,0 +1,29 @@
+package net.horizonsend.ion.server.features.client
+
+import net.starlegacy.util.Tasks
+import org.bukkit.event.EventHandler
+import org.bukkit.event.Listener
+import org.bukkit.event.player.PlayerJoinEvent
+import org.bukkit.event.player.PlayerQuitEvent
+import java.util.*
+
+class VoidNetwork : Listener {
+	@EventHandler
+	fun handshakeUser(event: PlayerJoinEvent) {
+		Tasks.syncDelay(10) {
```

Comment:
Why delay?

Reply:
We need to give the user some time to finish everything else it causes problems

Reply:
this isnt a reliable fix though

Reply:
plus i fixed the rei thingy that broke this so technically I can remove it but if it works

Reply:
any "fix" that relies on waiting an arbitrary amount of time is not a fix, its a hack